libqtile/backend/wayland/inputs.py

```diff
@@ -272,13 +275,23 @@ def _on_key(self, _listener: Listener, event: KeyboardKeyEvent) -> None:
 
         self.seat.keyboard_notify_key(event)
 
+    def _set_num_caps_lock(self, numlock: bool | None, capslock: bool | None) -> None:
+        """Set the initial num/caps lock state of the keyboard."""
+        mask = ModifiersMask(self.keyboard)
+        if numlock:
+            mask.add("Mod2")
```

Comment:
is it true that `Mod2` is always numlock? At least on x11, you'd need to query the modmap to see which thing is numlock, and add that one.

Reply:
I am not sure, maybe if the keys where remapped it will be different.
I don't really understand how xkb work but there's a modifier named `NumLock` does it guarantee that it's always numlock?

Reply:
I don't think it'll work if you put `NumLock` in there, but I honestly don't know. I think you probably have to do a xkb GetMap request to really get this to work right, which will require some changes in xcffib.